assets.md

```diff
@@ -37,6 +37,7 @@ This file is intended to list all the assets controlled by OpenTelemetry.
   * [Easy CLA](#easy-cla)
   * [Docker Hub](#docker-hub)
   * [OpenTelemetry Bot](#opentelemetry-bot)
+  * [otelbot](#otelbot)
```

Comment:
Given that you want folks to favor the `otelbot`, let's put that section ahead of the OTel bot (change it here and in the prose). WDYT?

```suggestion
  * [`otelbot`](#otelbot)
  * [OpenTelemetry Bot](#opentelemetry-bot)
```